### PR TITLE
Feature/split search columns by language

### PR DESCRIPTION
**Pull request by juuso-j**
# Models that are included in search now, has a search_column for every language.

#### Requirements.
1. requirements.in
2. requirements.txt
    * Uses mungeo 2.67 that has multilingual search_columns
 #### Api
1. services/api.py
    * Exclude the search_columns
2. services/search/api.py
    * Now uses the search_column for the language given as query param.
#### Indexing
1. services/management/commands/index_search_columns.py
2. services/signals.py
    * Updated to populate the multilingual search_columns
#### Models
1. services/models/service.py
3. services/models/unit.py
   * Replaced search_column with search columns for every language and indexes. Also changed the get_search_column_indexing to return indexing configuration for a given language

#### Migrations
1. services/migrations/0087_create_service_names_fields.py
    * Changed dependency
4. services/migrations/0088_create_multilingual_search_columns.py
    * Created the multilingual search columns and indexes
5. services/migrations/0089_create_search_view.py
    * Updated view with search_columns of every language.
 6. services/migrations/0090_trigram_index_service_and_unit_names.py
     * Updated dependency
#### Test
1. services/search/tests/conftest.py
   * Calls the get_search_column now with a language as argument.

### CODE REVIEW

File: services/models/unit.py [hunk 1]
```diff
@@ -267,24 +270,37 @@ def highlight_names(self):
...
+                ("address_zip", None, "D"),
+                ("street_address_fi", "finnish", "D"),
+            ]
+        elif lang == "sv":
+            return [
+                ("name_sv", "swedish", "A"),
+                ("service_names_sv", "swedish", "B"),
+                ("extra", None, "C"),
+                ("address_zip", None, "D"),
+                ("service_names_sv", "swedish", "B"),
```
Comment (mhieta) [inline services/models/unit.py:292]: This should be most probably fixed from `service_names_sv ` to `street_address_sv `

File: services/models/unit.py [hunk 2]
```diff
@@ -267,24 +270,37 @@ def highlight_names(self):
...
+            ]
+        elif lang == "en":
+            return [
+                ("name_en", "english", "A"),
+                ("service_names_en", "english", "B"),
+                ("extra", None, "C"),
+                ("address_zip", None, "D"),
+                ("service_names_en", "english", "B"),
```
Comment (mhieta) [inline services/models/unit.py:300]: This should be most probably fixed from `service_names_en ` to `street_address_en`

File: services/management/commands/index_search_columns.py [hunk 3]
```diff
@@ -30,15 +29,18 @@ def get_search_column(model):
...
-        )
+        for lang in ["fi", "sv", "en"]:
+            key = "search_column_%s" % lang
+            logger.info(
+                f"{lang} Units indexed: {Unit.objects.update(**{key: get_search_column(Unit, lang)})}"
+            )
+            logger.info(
+                f"{lang} Services indexed: {Service.objects.update(**{key: get_search_column(Service, lang)})}"
+            )   
+            adm_str = f"{lang} AdministrativeDivisions indexed: "
```
Comment (mhieta) [inline services/management/commands/index_search_columns.py:40]: This `adm_str` variable could be removed or simplified for code readability.

File: services/models/unit.py (hunk 1 above)
Comment (juuso-j) [reply services/models/unit.py]: True, fixed.

File: services/models/unit.py (hunk 2 above)
Comment (juuso-j) [reply services/models/unit.py]: Yes, fixed

**Review by mhieta — COMMENTED** [review]
Left couple of comments

File: services/management/commands/index_search_columns.py (hunk 3 above)
Comment (juuso-j) [reply services/management/commands/index_search_columns.py]: Fixed, by removing the variable. now:
     ```
    logger.info(
                f"{lang} AdministrativeDivisions indexed: "
                f"{AdministrativeDivision.objects.update(**{key: get_search_column(AdministrativeDivision, lang)})}"
            )
```

**Review by aceViilee — APPROVED** [review]

**Pull request merged**